packages/api-sdk/src/resources/projectSecret.ts

```diff
@@ -16,6 +16,12 @@ class ProjectSecretResource extends BaseResource<ProjectSecretResource> {
     return data;
   }
 
+  public async findManyByProjectID(projectID: string, tags: number[]) {
+    const tagsQuery = tags.join(',');
```

Comment:
```suggestion
    const tagsQuery = encodeURIComponent(tags.join(','));
```

Reply:
These really should be `URLSearchParam`s, instead of us having to manually encode them.

Reply:
I added a PR to support this: https://github.com/voiceflow/libs/pull/399